### ISSUE TITLE
Missing "clean" PHP FPM Debug images

### ISSUE THREAD

**Issue by bap14** [swiftotter/den#68]
## Problem
Using a pure PHP container without a defined Node version fails to locate the related `php-fpm-debug` image

## Steps to Reproduce
1. Initialize an environment `den env-init demo symfony`
2. Edit environment `.env` and remove `NODE_VERSION` value: `NODE_VERSION=`
3. Start environment: `den env up`
4. Encounter error:
```
Error response from daemon: failed to resolve reference "ghcr.io/swiftotter/den-php-fpm-debug:8.1": ghcr.io/swiftotter/den-php-fpm-debug:8.1: not found
```

## Expected Result
Being able to use and run pure PHP FPM Debug containers without Node installed

**Comment by ihor-sviziev** [comment]
@bap14 i think it's not very important since we have pre defined values in .env file by default.
Also, what is the reason not to use any of supported versions?

**Comment by navarr** [comment]
@ihor-sviziev I did intentionally add the functionality to disable node, but did not add images.  While I don't consider this high priority since the workaround of defining a node version is not a big issue, I would still like to resolve it at some point.

If one does not need node, why have those hundreds of Megabytes.

**Comment by bap14** [comment]
@ihor-sviziev Not every PHP project requires the use of Node, and so if it's not necessary, and the option to specify a node version is there, having a "clean" image with no node seems reasonable. Was just something I noticed as I was playing around with getting a Drupal install going (since I don't need Node, I just emptied the variable and saw there was no related debug image built (e.g. there was "php-fpm:8.1" but no "php-debug:8.1").
